src/components/toggle/style.css

```diff
@@ -0,0 +1,37 @@
+:host{
+	display: inline-block;
+	border: none;
+}
+button{
+	border: solid white 0.5px;
+	display: flex;
+	padding: 0.6em;
+	justify-content: center;
+	font-size: 110%;
+	font-weight: 400;
+	color: gray;
+	background-color: rgb(var(--smoothly-default-shade));
+}
```

Comment:
i've heard that we should aim to use smoothly colors... maybe we can do that here?